src/test/java/codeu/service/BBCodeParserTest.java

```diff
@@ -0,0 +1,64 @@
+package codeu.service;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BBCodeParserTest {
+
+  @Test
+  public void testParse_Empty() {
+    String input = "";
+    String expected = "";
+    String actual = BBCodeParser.parse(input);
+
```

Comment:
I would remove these newlines as well.  I would actually just make this one line

assertEquals("", BBCodeParser.parse(""));

Same goes for the rest of the tests.

Reply:
Oh nevermind, I guess that answers my other questions, I'll remove all the newlines.